### DISCUSSION TITLE
Doris BE running ERROR: INTERNAL_ERROR]Could not find local receiver for node 1 with instance 477cdcce8524a5e-b4c7790789106f85

### DISCUSSION THREAD

**Discussion by zhangtao106** [apache/doris#37254] [A - General / Q&A]
FE node running ok, but found error in BE node, message detail as this
W20240704 09:33:30.442440 53093 status.h:412] meet error status: [INTERNAL_ERROR]Could not find local receiver for node 1 with instance 477cdcce8524a5e-b4c7790789106f85

        0#  doris::vectorized::VDataStreamMgr::find_recvr(doris::TUniqueId const&, int, std::shared_ptr<doris::vectorized::VDataStreamRecvr>*, bool) at /var/local/ldb_toolchain/bin/../lib/gcc/x86_64-linux-gnu/11/../../../../include/c++/11/bits/basic_string.h:187
        1#  doris::vectorized::VDataStreamMgr::transmit_block(doris::PTransmitDataParams const*, google::protobuf::Closure**) at /var/local/ldb_toolchain/bin/../lib/gcc/x86_64-linux-gnu/11/../../../../include/c++/11/bits/unique_ptr.h:360
        2#  doris::PInternalServiceImpl::_transmit_block(google::protobuf::RpcController*, doris::PTransmitDataParams const*, doris::PTransmitDataResult*, google::protobuf::Closure*, doris::Status const&) at /home/zcp/repo_center/doris_release/doris/be/src/common/status.h:377
        3#  doris::PInternalServiceImpl::transmit_block(google::protobuf::RpcController*, doris::PTransmitDataParams const*, doris::PTransmitDataResult*, google::protobuf::Closure*) at /home/zcp/repo_center/doris_release/doris/be/src/service/internal_service.cpp:1506
        4#  brpc::policy::ProcessRpcRequest(brpc::InputMessageBase*)
        5#  brpc::ProcessInputMessage(void*)
        6#  brpc::InputMessenger::InputMessageClosure::~InputMessageClosure()
        7#  brpc::InputMessenger::OnNewMessages(brpc::Socket*)
        8#  brpc::Socket::ProcessEvent(void*)
        9#  bthread::TaskGroup::task_runner(long)
        10# bthread_make_fcontext

please helpe me ,thank you

**Comment by ixzc** [comment]
what did you do before your BE node core? which version of your doris? and please provide more error information

**Reply by zhangtao106** [reply]
my doris version is 2.1.3. this error is a whole group information.

**Reply by zhangtao106** [reply]
I was synchronize data from DRDS to doris by datax, i don't do other operations.

**Reply by fkzhao** [reply]
I also met this error

**Comment by bymeycc** [comment]
I also encountered the same problem, is there any solution?

**Comment by liujie-yt** [comment]
Me too, my Doris version is  2.1.4 
```
0# doris::vectorized::VDataStreamMgr::find_recvr(doris::TUniqueId const&, int, std::shared_ptr<doris::vectorized::VDataStreamRecvr>*, bool) at /var/local/ldb_toolchain/bin/../lib/gcc/x86_64-linux-gnu/11/../../../../include/c++/11/bits/basic_string.h:187
1# doris::vectorized::VDataStreamMgr::transmit_block(doris::PTransmitDataParams const*, google::protobuf::Closure**) at /var/local/ldb_toolchain/bin/../lib/gcc/x86_64-linux-gnu/11/../../../../include/c++/11/bits/unique_ptr.h:360
2# doris::PInternalServiceImpl::_transmit_block(google::protobuf::RpcController*, doris::PTransmitDataParams const*, doris::PTransmitDataResult*, google::protobuf::Closure*, doris::Status const&) at /home/zcp/repo_center/doris_release/doris/be/src/common/status.h:377
3# doris::PInternalServiceImpl::transmit_block(google::protobuf::RpcController*, doris::PTransmitDataParams const*, doris::PTransmitDataResult*, google::protobuf::Closure*) at /home/zcp/repo_center/doris_release/doris/be/src/service/internal_service.cpp:1523
4# brpc::policy::ProcessRpcRequest(brpc::InputMessageBase*)
5# brpc::ProcessInputMessage(void*)
6# brpc::InputMessenger::InputMessageClosure::~InputMessageClosure()
7# brpc::InputMessenger::OnNewMessages(brpc::Socket*)
8# brpc::Socket::ProcessEvent(void*)
9# bthread::TaskGroup::task_runner(long)
10# bthread_make_fcontext
RuntimeLogger W20250116 08:42:08.660655 2435 status.h:412] meet error status: [INTERNAL_ERROR]Could not find local receiver for node 1 with instance eafe031fc9a44f1-8c563e5f21f0f390
```

**Comment by ixzc** [comment]
we have fixed this, pr https://github.com/apache/doris/pull/36994
